app/dashboard/views/alias_contact_manager.py

```diff
@@ -44,6 +62,49 @@ def _check(form, field):
     return _check
 
 
+def create_contact(
+    user: User, alias: Alias, contact_address: str, fail_if_already_exist: bool = True
+) -> Contact:
+    if not contact_address:
+        raise ErrAddressInvalid("Empty address")
+    try:
+        contact_name, contact_email = parse_full_address(contact_address)
+    except ValueError:
+        raise ErrAddressInvalid(contact_address)
+
+    if not is_valid_email(contact_email):
```

Comment:
should we need to sanitize the contact address here? This is necessary as sometimes this address can contain artefacts due to copy/paste.

Reply:
Good idea!